app/src/main/java/com/example/peterchu/watplanner/home/HomePresenter.java

```diff
@@ -171,7 +171,7 @@ private void loadCourseCards() {
     }
 
     public void onCalendarEventClicked(CourseComponent courseComponent) {
-        List<List<CourseComponent>> alternatives;
+        Set<List<CourseComponent>> alternatives;
```

Comment:
I like the idea, but using a `set` doesn't help preserve the natural ordering of the `List` - which will come in handy when reordering the calendar. 

suggestion - maybe a `LinkedSet`?